src/components/tooltip/Tooltip.tsx

```diff
@@ -217,8 +219,27 @@ class Tooltip extends React.Component<TooltipProps, State> {
             stopBubble,
             text,
             theme,
+            intl,
         } = this.props;
 
+        let ariaLabel;
+        let tooltipText;
+
+        if (text) {
+            if (typeof text === 'string') {
+                tooltipText = text;
+            } else if (React.isValidElement(text) && getProp(text, 'props')) {
+                const { id, defaultMessage, description, values } = getProp(text, 'props');
```

Comment:
I think this should be fine for the first pass (still wrapping my head around updating the child's `aria-label` or adding in a `describedby` label. Can we add comments in this conditional to explain that we're specifically looking for these props since they indicate a `FormattedMessage` component is passed into `text` so that it's clear?